pkg/routeagent_driver/handlers/mtu/mtuhandler.go

```diff
@@ -62,7 +62,7 @@ var logger = log.Logger{Logger: logf.Log.WithName("MTU")}
 
 func NewMTUHandler(localClusterCidr []string, isGlobalnet bool, tcpMssValue int) event.Handler {
 	forceMss := notNeeded
-	if isGlobalnet {
+	if isGlobalnet || tcpMssValue != 0 {
```

Comment:
Not in the scope of this PR, but we should probably add some validation on the MTU value that is configured by the user (i.e., to ensure that its in the valid reasonable range)